tests/cnf/core/network/internal/cmd/cmd.go

```diff
@@ -80,3 +82,81 @@ func GetSrIovPf(vfInterfaceName, namespace, nodeName string) (string, error) {
 
 	return strings.TrimRight(pfName, "\r\n"), nil
 }
+
+// RxTrafficOnClientPod verifies the incoming packets on the dpdk client pod from the dpdk server.
+func RxTrafficOnClientPod(clientPod *pod.Builder, clientRxCmd string) error {
+	timeoutError := "command terminated with exit code 137"
+
+	glog.V(90).Infof("Checking dpdk-pmd traffic command %s from the client pod %s",
+		clientRxCmd, clientPod.Definition.Name)
+
+	err := clientPod.WaitUntilRunning(time.Minute)
+
+	if err != nil {
+		return fmt.Errorf("failed to wait until pod is running with error %w", err)
+	}
+
+	clientOut, err := clientPod.ExecCommand([]string{"/bin/bash", "-c", clientRxCmd})
+
+	if err.Error() != timeoutError {
+		return fmt.Errorf("failed to run the dpdk-pmd command on the client pod %s with error %s",
+			clientRxCmd, clientOut.String())
+	}
+
+	// Parsing output from the DPDK application
+	glog.V(90).Infof("Processing testpdm output from client pod \n%s", clientOut.String())
+	outPutTrue := checkRxOnly(clientOut.String())
+
+	if !outPutTrue {
+		return fmt.Errorf("failed to parse the output from RxTrafficOnClientPod")
+	}
+
+	return nil
+}
+
+// checkRxOnly checks the number of incoming packets.
+func checkRxOnly(out string) bool {
+	lines := strings.Split(out, "\n")
+	for index, line := range lines {
+		if strings.Contains(line, "NIC statistics for port") {
+			if len(lines[index+1]) < 3 {
+				fmt.Println("Fail: line list contains less than 3 elements")
```

Comment:
Please use glog insted for all fmt.Println

Reply:
changed